### ISSUE TITLE
Hyrax::Callbacks::NotEnabled on FetchGraphWorker

### ISSUE THREAD

**Issue by luisgreg99** [OregonDigital/OD2#983]
### Descriptive summary

When trying to QA https://github.com/OregonDigital/hyrax-migrator/issues/115 on staging with migration, I found an issue that might be related to a recent update https://github.com/OregonDigital/OD2/pull/974

Error backtrace:

```
2020-01-31T22:28:53.015Z pid=1 tid=ov0sat4rd WARN: {"context":"Job raised exception","job":{"retry":11,"queue":"default","class":"FetchGraphWorker","args":["fx71bd22d","admin@example.org"],"jid":"881fbc314c14ce81b5b6aa2b","created_at":1580509331.5896242,"enqueued_at":1580509731.9233837,"error_message":"Hyrax::Callbacks::NotEnabled","error_class":"Hyrax::Callbacks::NotEnabled","failed_at":1580509332.8881867,"retry_count":3,"retried_at":1580509544.8344445},"jobstr":"{\"retry\":11,\"queue\":\"default\",\"class\":\"FetchGraphWorker\",\"args\":[\"fx71bd22d\",\"admin@example.org\"],\"jid\":\"881fbc314c14ce81b5b6aa2b\",\"created_at\":1580509331.5896242,\"enqueued_at\":1580509731.9233837,\"error_message\":\"Hyrax::Callbacks::NotEnabled\",\"error_class\":\"Hyrax::Callbacks::NotEnabled\",\"failed_at\":1580509332.8881867,\"retry_count\":3,\"retried_at\":1580509544.8344445}"}
2020-01-31T22:28:53.015Z pid=1 tid=ov0sat4rd WARN: Hyrax::Callbacks::NotEnabled: Hyrax::Callbacks::NotEnabled
2020-01-31T22:28:53.015Z pid=1 tid=ov0sat4rd WARN: /usr/local/bundle/bundler/gems/hyrax-da1b6ffd7c0e/lib/hyrax/callbacks/registry.rb:38:in `run'
/data/app/workers/fetch_graph_worker.rb:58:in `fetch_failed_callback'
/data/app/workers/fetch_graph_worker.rb:27:in `rescue in block (2 levels) in perform'
/data/app/workers/fetch_graph_worker.rb:24:in `block (2 levels) in perform'
/usr/local/bundle/gems/active-triples-1.1.0/lib/active_triples/relation.rb:323:in `block in each'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/enumerable.rb:482:in `block in each_object'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/queryable.rb:291:in `<<'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/queryable.rb:291:in `block (2 levels) in enum_for'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:415:in `block (4 levels) in query_pattern'
/usr/local/bundle/gems/hamster-3.0.0/lib/hamster/trie.rb:40:in `block in each'
/usr/local/bundle/gems/hamster-3.0.0/lib/hamster/trie.rb:40:in `each'
/usr/local/bundle/gems/hamster-3.0.0/lib/hamster/trie.rb:40:in `each'
/usr/local/bundle/gems/hamster-3.0.0/lib/hamster/hash.rb:331:in `each'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:413:in `block (3 levels) in query_pattern'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:412:in `each'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:412:in `block (2 levels) in query_pattern'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:404:in `each'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:404:in `block in query_pattern'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:392:in `each'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/repository.rb:392:in `query_pattern'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/queryable.rb:291:in `block in enum_for'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/enumerable.rb:159:in `each'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/enumerable.rb:159:in `each'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/enumerable.rb:159:in `each_statement'
/usr/local/bundle/gems/rdf-3.1.1/lib/rdf/mixin/enumerable.rb:478:in `each_object'
/usr/local/bundle/gems/active-triples-1.1.0/lib/active_triples/relation.rb:487:in `objects'
/usr/local/bundle/gems/active-triples-1.1.0/lib/active_triples/relation.rb:321:in `each'
/data/app/workers/fetch_graph_worker.rb:21:in `block in perform'
/data/app/workers/fetch_graph_worker.rb:20:in `each'
/data/app/workers/fetch_graph_worker.rb:20:in `perform'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:196:in `execute_job'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:164:in `block (2 levels) in process'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/middleware/chain.rb:133:in `invoke'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:163:in `block in process'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:136:in `block (6 levels) in dispatch'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/job_retry.rb:111:in `local'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:135:in `block (5 levels) in dispatch'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/rails.rb:43:in `block in call'
/usr/local/bundle/gems/activesupport-5.1.7/lib/active_support/execution_wrapper.rb:85:in `wrap'
/usr/local/bundle/gems/activesupport-5.1.7/lib/active_support/reloader.rb:68:in `block in wrap'
/usr/local/bundle/gems/activesupport-5.1.7/lib/active_support/execution_wrapper.rb:85:in `wrap'
/usr/local/bundle/gems/activesupport-5.1.7/lib/active_support/reloader.rb:67:in `wrap'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/rails.rb:42:in `call'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:131:in `block (4 levels) in dispatch'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:257:in `stats'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:126:in `block (3 levels) in dispatch'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/job_logger.rb:13:in `call'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:125:in `block (2 levels) in dispatch'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/job_retry.rb:78:in `global'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:124:in `block in dispatch'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/logger.rb:10:in `with'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/job_logger.rb:33:in `prepare'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:123:in `dispatch'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:162:in `process'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:78:in `process_one'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/processor.rb:68:in `run'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/util.rb:15:in `watchdog'
/usr/local/bundle/gems/sidekiq-6.0.4/lib/sidekiq/util.rb:24:in `block in safe_thread'
```

### Expected behavior
The work got ingested successfully, but this job `FetchGraphWorker` got stuck on retries.

### Related work

Probably related to https://github.com/OregonDigital/OD2/pull/974

### Accessibility Concerns

NA

**Comment by straleyb** [comment]
Fixed by #984

**Comment by luisgreg99** [comment]
QA Pass!
